angular-frontend/_app/templates/login.template.html

```diff
@@ -1,359 +1,61 @@
 <style>
-    /*------------------------------------------------------------------
-    [Table of contents]
-
-    1. General Structure
-    2. Anchor Link
-    3. Text Outside the Box
-    4. Main Form
-    5. Login Button
-    6. Form Invalid
-    7. Form - Main Message
-    8. Custom Checkbox & Radio
-    9. Misc
-    -------------------------------------------------------------------*/
-    /*=== 2. Anchor Link ===*/
-    a {
-    color: #aaaaaa;
-    transition: all ease-in-out 200ms;
-    }
-    a:hover {
-    color: #ffffff;
-    text-decoration: none;
-    }
-    /*=== 3. Text Outside the Box ===*/
-    .etc-login-form {
-    color: #919191;
-    padding: 10px 20px;
-    }
-    .etc-login-form p {
-    margin-bottom: 5px;
-    }
-    /*=== 4. Main Form ===*/
-    .login-form-1 {
-    max-width: 300px;
-    border-radius: 5px;
-    display: inline-block;
-    }
-    .main-login-form {
-    position: relative;
-    }
-    .login-form-1 .form-control {
-    border: 0;
-    box-shadow: 0 0 0;
-    border-radius: 0;
-    background: transparent;
-    color: #555555;
-    padding: 7px 0;
-    font-weight: bold;
-    height:auto;
-    }
-    .login-form-1 .form-control::-webkit-input-placeholder {
-    color: #999999;
-    }
-    .login-form-1 .form-control:-moz-placeholder,
-    .login-form-1 .form-control::-moz-placeholder,
-    .login-form-1 .form-control:-ms-input-placeholder {
-    color: #999999;
-    }
-    .login-form-1 .form-group {
-    margin-bottom: 0;
-    border-bottom: 2px solid #efefef;
-    padding-right: 20px;
-    position: relative;
-    }
-    .login-form-1 .form-group:last-child {
-    border-bottom: 0;
-    }
-    .login-group {
-    background: #ffffff;
-    color: #999999;
-    border-radius: 8px;
-    padding: 10px 20px;
-    }
-    .login-group-checkbox {
-    padding: 5px 0;
+    input {
```

Comment:
Lets try and stick with the Material Design defaults as much as we can

Or if we're deviating, lets try and add things into a separate .css or .less file that we can reference